core/new-gui/src/app/workspace/service/execute-workflow/execute-workflow.service.ts

```diff
@@ -170,6 +170,11 @@ export class ExecuteWorkflowService {
     }, FORM_DEBOUNCE_TIME_MS);
     this.updateExecutionState({ state: ExecutionState.WaitingToRun });
     this.setExecutionTimeout('submit workflow timeout', ExecutionState.Running, ExecutionState.Failed);
+
+    // add flag for new execution of workflow
+    // so when next time the result panel is displayed, it will use new data
+    // instead of those stored in the session storage
+    sessionStorage.setItem('newWorkflowExecuted', 'true');
```

Comment:
make the key more specific, and also put the key in a separate variable (public static), so we are keep tracking of what we are storing in sessionstorage / localstorage

Reply:
Updated in https://github.com/Texera/texera/pull/903/commits/d668ef2d6eb6c5ef694c00eeaa984ccf51bd860e

Looks like putting the key in the ResultPanelComponent class as a public static member will lead to circular dependency
(ResultPanelComponent -> ExecuteWorkflowService-> ResultPanelComponent). I put the key into `result-table.interface.ts`. Maybe there's a better way to solve this issue?